modules/installing-rhacs-kubernetes-steps.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-high-level-overview
+:_mod-docs-content-type: CONCEPT
+[id="installing-rhacs-kubernetes-steps_{context}"]
+= Installation steps for RHACS on Kubernetes
+
+[id="installing-kube-helm-steps"]
+== Installing {product-title-short} on Kubernetes platforms by using Helm charts
+
+. Add the {product-title-short} Helm charts repository.
+. Install the `central-services` Helm chart on the cluster that will contain Central, called the Central cluster.
+. Log in to the {product-title-short} web console from the Central cluster and create an init bundle that you will install on the cluster that you want to secure, called the secured cluster.
+. For each secured cluster:
+.. Apply the init bundle you created with {product-title-short}. Log in to the secured cluster and run the `kubectl create -f <init_bundle>.yaml -n <stackrox>` command, specifying the path to the downloaded YAML file of the init bundle.
+.. Install the `secured-cluster-services` Helm chart on the secured cluster, specifying the path to the init bundle that you created earlier.
+
+[id="installing-kube-roxctl-steps"]
+== Installing {product-title-short} on Kubernetes platforms by using the `roxctl` CLI
+
+This installation method is also called the _manifest installation method_.
```

Comment:
The _manifest installation method_ phrase is used a lot in this PR. Are they all the same process?

Reply:
That makes sense now. Thanks!